### DISCUSSION TITLE
Ideas on expanding WZDx to include events outside of work zones?

### DISCUSSION THREAD

**Discussion by hua-xiang** [usdot-jpo-ode/jpo-wzdx#104] [Ideas]
One interesting thing we in Maryland are talking about internally is on the “streeteries,” i.e. when local jurisdictions close our state highway (when it’s their “main street”) to accommodate outdoor dining needs as encouraged by our Governor, how can we get those closures in our ATMS traffic event feeds to inform drivers. Any similar discussions in your state?

**Comment by RHolt-RIDOT** [comment]
No discussion/experiences with data/feeds yet on this peculiarity from me/my little DOT (Rhode Island), but good question...      Perhaps just treating as complete Road Closure will do the trick?  This seems to me to be an example of where the WZDx may really be doing more than it's name implies (i.e., sufficient to cover not just Work Zone but also PSE (Planned Special Event) related closures, which I would argue your "streeteries" example really is...  If the V2 specification is not yet robust enough to cover this, then at least to me ideally V3 would be.

**Comment by rdsheckler** [comment]
I think that the issue here lies in what is defined as a work zone.  Does closing a road (or lane) due to seasonal, weather, of flooding issues fall into the WZDX effort?  Does utility or first responder activity fall into the frame work?  If the WZDX is to be used for navigation it follows that these issues must be included and this effort must be significantly expanded.

**Comment by eli104** [comment]
Actually a very interesting "problem," which may skew the definitions of the Work Zone group... But many large and small cities are reducing lanes or closing streets on a regular (scheduled) basis for any number of reasons (outdoor dining, or even Safe Streets programs to promote pedestrian and cycle traffic). So perhaps instead of asking, "Are these Work Zones?" We might consider, "Should the WZDx effort cover ANY medium to long term closures which are NOT related to emergency response (e.g. crashes)?" 

I am of the opinion that while we don't need to reconsider what a Work Zone is, we might consider re-defining a Planned Closure. Specifically, something like a roadway or lane closure which has been planned and agreed by the presiding roadway agency, which limits vehicular access to or on the road." (Sorry... maybe that's too formal). These include (but are not limited to):
- Lanes or roads which are permanently closed for outdoor activities (dining, etc.).
- Lanes or roads which are temporarily closed on a REGULAR and RECURRENT basis for outdoor activities (dining, pedestrian/cycle access, etc.), but NOT events that occur less than one day each week. (This removes things like annual road races or summer weekend festivals from the mix). 
- Planned construction activities (the stuff we are already discussing).

Side note: There are already many areas where traffic and navigation providers are upping their coverage of Safe Streets-like activities in cities across the US and Europe (likely other areas). These have become far more prevalent since the pandemic, and though some like outdoor dining is more focused on warmer weather, there are enough semi-permanent structures (i.e. wheeled partitions) into which portable heaters have been used in the early spring chill.

**Comment by rdsheckler** [comment]
The danger in expanding the purpose of the WZDX spec is that it rapidly
gets overwhelmed by other things like utility work.  Those activities
outnumber 'work zones', and aside from some of us data generators and some
consumers it is a whole new set of contributors who will drive the spec to
places that the agencies don't want to go.

The unspoken reality is that 90% of the temporary events do not involve the
states or the big municipalities.
